File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Address.java
package com.babestudios.companyinfouk.data.model.persons;


import com.google.gson.annotations.SerializedName;

public class Address {
	@SerializedName("country")
	public String country;
	@SerializedName("postal_code")
	public String postalCode;
	@SerializedName("address_line_1")
	public String addressLine1;
	@SerializedName("locality")
	public String locality;
	@SerializedName("region")
	public String region;
	@SerializedName("premises")
	public String premises;
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/persons/Address.kt
package com.babestudios.companyinfouk.data.model.persons


import com.google.gson.annotations.SerializedName

class Address {
	@SerializedName("country")
	var country: String? = null
	@SerializedName("postal_code")
	var postalCode: String? = null
	@SerializedName("address_line_1")
	var addressLine1: String? = null
	@SerializedName("locality")
	var locality: String? = null
	@SerializedName("region")
	var region: String? = null
	@SerializedName("premises")
	var premises: String? = null
}
